domain/src/main/java/com/example/domain/usecase/GetHeartRateDataUseCase.kt

```diff
@@ -6,25 +6,24 @@ import kotlinx.coroutines.flow.Flow
 
 class GetHeartRateDataUseCase(private val repository: IHeartRateHistoryRepository) {
```

Comment:
The name of use case is a little bit confusing now since you not only getting heart rate here